commerce_coordinator/apps/lms/views.py

```diff
@@ -401,3 +413,148 @@ def post(self, request):
             logger.exception(f"[FirstTimeDiscountEligibleView] HTTP Error: {err}")
 
         return Response({'is_eligible': True})
+
+
+class ProgramPriceView(APIView):
+    """View to get the price of a program."""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (AllowAny,)
```

Comment:
As this call will be made by LMS via a admin user.
```suggestion
    permission_classes = (IsAdmin,)
```

Reply:
This call will also be made by the marketing site for program about page and user can be unauthenticated so it should `AllowAny`

Reply:
We checked on legacy ecommerce the permission class is `IsAuthenticated`. We are adding `IsAuthenticated` here too.